packages/fast-web-utilities/src/dom.ts

```diff
@@ -5,3 +5,24 @@
 export function getKeyCode(event: KeyboardEvent): number {
     return (event === null) ? null : event.which || event.keyCode || event.charCode;
 }
+
+/**
+ * Check if an element is a descendent of another element.
+ */
+export function isDescendant(parent: (HTMLElement | Element), child: (HTMLElement | Element)): boolean {
+    if (!parent || !child) {
+        return false;
+    }
+
+    let node: Node = child.parentNode;
+
+    while (!!node) {
+        if (node === parent) {
+            return true;
+        }
+
+        node = node.parentNode;
+    }
+
+    return false;
+}
```

Comment:
I see - This was intentionally removed for element.contains. We should be able to accomplish this without this function.